### PR TITLE
Littlebird hud update

### PR DESCRIPTION
**Pull request by Seb105**
- Use currentPilot
- Improved respecting of "cnto_littlebird_hud_enable"

### CODE REVIEW

File: addons/littlebird_hud/fn_showGreaseDot.sqf [hunk 1]
```diff
@@ -0,0 +1,9 @@
+params ["_vehicle", "_enable"];
+
+if (_vehicle isKindOf "RHS_MELB_AH6M") then {
+        private _status = [0, 1] select _enable;
+        _vehicle animateSource ["Addcrosshair", _status, true];
+    } else {
+        private _status = ["", "a3\air_f\heli_light_01\data\heli_light_01_dot_ca.paa"] select _enable;
```
Comment (freghar) [inline addons/littlebird_hud/fn_showGreaseDot.sqf:7]: Btw, an alternate way of writing this that's closer to what other languages use (as a ternary operator) is
```sqf
private _status = if (_enable) then { "" } else { "a3\air_f\heli_light_01\data\heli_light_01_dot_ca.paa" };
```
Either way is fine, though.

**Comment by freghar** [comment]
This PR looks good to me, as long as it's been tested to work.

**Comment by Seb105** [comment]
Works in SP, will test it in MP this saturday and post findings here.

**Comment by Seb105** [comment]
Tested in MP, works.

**Review by milivojm — APPROVED** [review]

**Pull request merged**